packages/ns-phonehome/files/phonehome

```diff
@@ -74,17 +74,25 @@ if not product:
     except:
         product = ""
 
+nethsec_version = ""
+version = ""
 with open('/etc/os-release', 'r') as file:
     for line in file:
-        if line.startswith("VERSION_ID"):
+        if line.startswith("VERSION_ID="):
             nethsec_version = line.split('=')[1].split('-')[2][3:]
             break
```

Comment:
```suggestion
            
```

Reply:
Well, it works already because VERSION comes before VERSION_ID in os_release file, but I agree to fix it.

Reply:
I know how the `os-release` file is made, however we've seen broken configuration files in packages from upstream, everything can happen :upside_down_face: